connector/interfaces/work_order_connector.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+
+
+class WorkOrderConnector(ABC):
+    """
+    WorkOrderRegistryInterface is an abstract base class that contains
```

Comment:
Looks like name and the class names do not match for these connectors (even the work order receipt, the worker register connector etc...)

Reply:
The naming mismatch also occurs in other files. Because class names are often renamed, do not repeat the name in the class doc string.  Use something like: "This class is ..." instead here and elsewhere.

Reply:
Done